WMass/python/plotter/wmassUtilities/theory.py

```diff
@@ -3,7 +3,7 @@
         "name" : "NNPDF31",
         "nhessian" : 100, # basically number of actual PDF hessians (no nominal and no alpha if any)
         "onlyW" : True, # Because the alpha_s sets are missing
-        "truncate" : False,
+        "truncate" : True, # for nnpdf31 this is for not using alpha from the last two bins, stopping after the hessians
```

Comment:
If you truncate it why do you need the offsets in the mirrorGroups later?

Reply:
The offset in MirrorGroup is used so that when the text label is built from an "i,label in enumerate()" loop as str(i) it will start from 1 and thus have pdf1 for first hessian